bin/entrypoint.sh

```diff
@@ -8,14 +8,7 @@ USER_ID=${LOCAL_USER_ID:-999}
 
 id -u odoo &> /dev/null || useradd --shell /bin/bash -u $USER_ID -o -c "" -m odoo
 
-confd -log-level=warn -onetime -backend ${CONFD_BACKEND:-env} ${CONFD_OPTS:-}
-
-# TODO this could (should?) be sourced from file(s) under confd control
-export PGHOST=${DB_HOST}
-export PGPORT=${DB_PORT:-5432}
-export PGUSER=${DB_USER}
-export PGPASSWORD=${DB_PASSWORD}
-export PGDATABASE=${DB_NAME}
+source /odoo/templates/answers.sh && envsubst < /odoo/templates/odoo.cfg.tmpl > /etc/odoo.cfg
```

Comment:
@sbidoul  confd replacement is here

Reply:
Yes, feel free to continue this route :)